### PR TITLE
refactor: Changes in platform email templates.

### PR DESCRIPTION
**Pull request by pranalidhanavade**
### What 

- Changes in following email templates
1.  User onboarding template.
2.  Organization invitation template.
3.  Ecosystem invitation template.
4.  Issuance email.
5.  Verification email.

### CODE REVIEW

File: apps/user/src/user.controller.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export class UserController {
 
   @MessagePattern({ cmd: 'get-user-by-mail' })
   async findUserByEmail(payload: { email }): Promise<object> {
-    return this.userService.findUserByEmail(payload);
```
Comment (nishad-ayanworks) [inline apps/user/src/user.controller.ts:73]: Please check if this function `findUserByEmail` is used somewhere before removing it.

**Review by nishad-ayanworks — CHANGES_REQUESTED** [review]
Please check comment

File: apps/user/src/user.controller.ts (hunk 1 above)
Comment (pranalidhanavade) [reply apps/user/src/user.controller.ts:73]: Resolved.

**Review by nishad-ayanworks — APPROVED** [review]
LG

**Review by KulkarniShashank — APPROVED** [review]
LG

**Pull request merged**